src/platform/elements/time-picker/TimePickerInput.ts

```diff
@@ -201,6 +201,11 @@ export class NovoTimePickerInputElement implements OnInit, ControlValueAccessor
       minute: '2-digit',
       hour12: !this.military,
     });
+    if (format.indexOf('vorm.') > -1) {
```

Comment:
Shouldn't have locale specific code here. It should go into the i18n files and passed in via that.